### PR TITLE
feat: create structure and basic event view

### PR DESCRIPTION
**Pull request by gularsson**
TODO as other issues:
Datetime handling, currently in progress
~Registration, questions raised about its placement i newsdoc~ probably keep it like it is?
Connect planning from event
Show expanded organiser information? How?
General looks

### CODE REVIEW

File: src/views/Event/index.tsx [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+            </div>
+
+            <Description role='public' />
+            <Registration />
+
+            <p>Datetime TODO</p>
+          </div>
+
+          <div className='flex flex-col space-y-2 w-fit'>
+            <div className='grid grid-cols-2 gap-2 @xs:flex'>
```
Comment (dannelundqvist) [inline src/views/Event/index.tsx:130]:
```suggestion
            <div className='flex flex-wrap gap-2'>
```

File: src/views/Event/index.tsx
Comment (dannelundqvist) [inline src/views/Event/index.tsx:1]: It is not possible to clear category.

**Review by dannelundqvist — CHANGES_REQUESTED** [review]
Really good work. Nice that we generalize many components to be more reusable now.

Two small issues in the actual event dialog though. Category combobox issue and a suggested improvement to make the comboboxes work better when screen estate get small.

File: src/views/Event/index.tsx (hunk 1 above)
Comment (gularsson) [reply src/views/Event/index.tsx:130]: Does that work for you? Cause that was my first try, but it didn't work as expected :thinking: 
I'll try again.

File: src/views/Event/index.tsx
Comment (gularsson) [reply src/views/Event/index.tsx:1]: :+1:

**Review by dannelundqvist — APPROVED** [review]

**Pull request merged**